src/SDKs/WebSites/Management.Websites/Generated/AppServiceCertificateOrdersOperations.cs

```diff
@@ -1,15 +1,15 @@
+// <auto-generated>
```

Comment:
The generated code looks fine, but:

(1) Please include the metadata file here:https://github.com/Azure/azure-sdk-for-net/tree/psSdkJson6/src/SDKs/_metadata   which is automatically generated by AutoRest
(2) Please update the csproj with release notes and a version update
(3) Please include tests

Reply:
(1) I can't find this metadata file anywhere after generating the code with AuotRest. I'm running "autorest --csharp" from azure-rest-api-specs/specification/web/resource-manager with autorest 2.0.4166. The only output I see is C# code. Any advice on how I can get the metadata?
(3) I'll include a test for the PremiumV2 SKU that was added, but other updates in the PR are just metadata changes. Recorded tests should still be passing. Hopefully that's good enough for now.

Reply:
@Nking92 if you do not have a generate.cmd under you sdk folder, you should create one by modelling it from other RPs.
Let me know if you cannot find the generate.cmd file which will generate your SDK

Reply:
Got it, I was trying to directly use autorest instead of using the generate.cmd script. I regenerated with that script in the latest commit. It's a large number of changes once again, but it is working fine with our PowerShell project, which is good news.

Reply:
I opened an issue to write more tests at https://github.com/Azure/azure-sdk-for-net/issues/3844. Our API is so large that I would not be able to test it all without a huge time investment. I realize tests important, but we are also under pressure to get an SDK update published, which we can use for PowerShell, which has some broken commands that are causing pain for customers.